client/src/components/_common/CustomTreeItem.tsx

```diff
@@ -75,12 +83,17 @@ const CustomContent = React.forwardRef(function CustomContent(
         [classes.focused]: focused,
         [classes.disabled]: disabled,
       })}
-      onMouseDown={handleMouseDown}
+      onMouseDown={handleInteractionEvent}
+      onKeyDown={(e) => {
+        if (e.key === 'Enter') handleInteractionEvent(e);
+      }}
       ref={ref as React.Ref<HTMLDivElement>}
     >
       <div
         onClick={handleExpansionAction}
-        onKeyPress={handleExpansionAction}
+        onKeyDown={(e) => {
+          if (e.key === 'Enter') handleExpansionAction(e);
```

Comment:
Was this change to only limit expanding menu items to trigger on Enter intentional? I see that previously I could hit space or any key but now it's only on Enter. Not sure if it matters from a 508 perspective or anything like that

Reply:
Yep, this is because expanding on any key interferes with tab-through navigation now.